### ISSUE TITLE
Add global default outbound

### ISSUE THREAD

**Issue by Arctomachine** [MatsuriDayo/NekoBoxForAndroid#810]
## Description suggestions
Add default outbound for global settings. Same as desktop version:
![image](https://github.com/user-attachments/assets/fb76c6fd-049c-4724-8de3-ec958752c354)
## Necessity of recommendations
https://github.com/MatsuriDayo/NekoBoxForAndroid/issues/723#issuecomment-2448282977 is only applied to one config, not globally. And it is not working like default outbound should: it overwrites all route rules instead of acting as fallback

**Comment by RAZUMNO** [comment]
Agree. Need that option so I can use "bypass" on everything except the specific domains I want to "proxy/vpn"
